src/evm.rs

```diff
@@ -368,14 +368,15 @@ impl EVM {
         }
     }
 
+    #[allow(clippy::useless_asref)]
```

Comment:
```suggestion
```

Not good smell to use those

Reply:
yeah but the suggestion given by clippy does not work (see above)